src/tracks/playabletrack/wavetrack/ui/WaveTrackAffordanceControls.cpp

```diff
@@ -33,13 +33,6 @@ class WaveTrackAffordanceHandle final : public AffordanceHandle
 public:
     WaveTrackAffordanceHandle(const std::shared_ptr<Track>& track) : AffordanceHandle(track) { }
 
-    static UIHandlePtr HitAnywhere(std::weak_ptr<AffordanceHandle>& holder, const std::shared_ptr<Track>& pTrack)
-    {
-        auto result = std::static_pointer_cast<AffordanceHandle>(std::make_shared<WaveTrackAffordanceHandle>(pTrack));
-        result = AssignUIHandlePtr(holder, result);
```

Comment:
This change is good, but incomplete!

Similar code with the same slicing error is still not corrected in NoteTrackAffordanceControls.

Reply:
Thanks! fixed